src/clojure_tic_tac_toe/game_handler.clj

```diff
@@ -0,0 +1,44 @@
+(ns clojure-tic-tac-toe.game_handler
+  (:require [clojure-tic-tac-toe.input_output :as io]
+            [clojure-tic-tac-toe.board :as board]))
+
+(def starting-game-state
+  {:board board/empty-board, :player :X, :finished? false})
+
+(defn- switch-player
+  [board player]
+  (let [tokens (keys board)]
+    (first
+      (remove #{player} tokens))))
+
+(defn- is-game-finished?
+  [board]
+  (empty? (board/get-available-moves board)))
+
+(defn play-round
+  [game-state]
+  (let [board  (:board  game-state)
+        player (:player game-state)
+        move (io/get-move board player)
+        next-player (switch-player board player)
+        updated-board (board/add-move board move player)
+        game-is-finished (is-game-finished? updated-board)]
+    (io/display-board updated-board)
+    (zipmap [:board :player :finished?]
```

Comment:
`zipmap` shouldn't be necessary.  You can just use a map literal.

Reply:
changed

Reply:
Don't just tell us it's done.  Push the commit that fixes it.